users/keppenj/index.md

```diff
@@ -4,4 +4,12 @@ layout: default
 author: Jackson Keppen
```

Comment:
@keppenj Sorry about this being so late, I didn't write it on my list.. your index needs to have this:

---
title: Jackson-K-pesonal
layout: default
author: Jackson Keppen
---
Jackson Keppen
================================